backend_new/src/services/user.service.ts

```diff
@@ -24,12 +41,21 @@ const view: Prisma.UserAccountsInclude = {
   userRoles: true,
 };
 
+type findByOptions = {
+  userId?: string;
+  email?: string;
+};
+
 @Injectable()
 export class UserService {
-  constructor(private prisma: PrismaService) {}
+  constructor(private prisma: PrismaService) {
+    dayjs.extend(advancedFormat);
+  }
 
   /*
     this will get a set of users given the params passed in
+    Only users with a user role of admin or jurisdictional admin can get the list of available users. 
+    This means we don't need to account for a user with only the partner role when it comes to accessing this function
```

Comment:
I could be missing it, but looks like this functionality wasn't added yet. Should we add a todo to make sure that is added?

Reply:
that is handled through auth, basically a partner role only user wouldn't be able to hit the user list end point